docs/source/how_to_guides/optimization/how_to_specify_algorithm_and_algo_options.rst

```diff
@@ -679,18 +679,24 @@ noisy criterion functions.
       Function argument to find_affine_points() for the case where input array
       *model_improving_points* is zero.
     - **c2** (int): Treshold for accepting the norm of our current x candidate.
-      Equal to 10 by default. Argument to find_affine_points() in case
+      Equal to 10 by default. Argument to *find_affine_points()* in case
       the input array *model_improving_points* is not zero.
     - **trustregion_subproblem_solver** (str): Solver to use for the trust-region
       subproblem. Two internal solvers are supported:
       - "bntr": Bounded Newton Trust-Region (default, supports bound constraints)
       - "gqtpar": (does not support bound constraints)
+    - **trustregion_subproblem_conjugate_gradient_step** (str): Routine for computing
```

Comment:
I think you forgot to adjust the documentation here. To be sure: `trustregion_subproblem_conjugate_gradient_step` should not be a separate argument but part of the `trustregion_subsolver_options` and called `conjugate_gradient_method` or `bntr_conjugate_gradient_method`.